.bingo/Variables.mk

```diff
@@ -0,0 +1,42 @@
+# Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.2.2. DO NOT EDIT.
+# All tools are designed to be build inside $GOBIN.
+GOPATH ?= $(shell go env GOPATH)
+GOBIN  ?= $(firstword $(subst :, ,${GOPATH}))/bin
+GO     ?= $(shell which go)
+
+# Bellow generated variables ensure that every time a tool under each variable is invoked, the correct version
+# will be used; reinstalling only if needed.
+# For example for bingo variable:
+#
+# In your main Makefile (for non array binaries):
+#
+#include .bingo/Variables.mk # Assuming -dir was set to .bingo .
+#
+#command: $(BINGO)
+#	@echo "Running bingo"
+#	@$(BINGO) <flags/args..>
+#
+BINGO := $(GOBIN)/bingo-v0.2.3
+$(BINGO): .bingo/bingo.mod
+	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
+	@echo "(re)installing $(GOBIN)/bingo-v0.2.3"
+	@cd .bingo && $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.2.3 "github.com/bwplotka/bingo"
+
+GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.27.0
+$(GOLANGCI_LINT): .bingo/golangci-lint.mod
+	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
+	@echo "(re)installing $(GOBIN)/golangci-lint-v1.27.0"
+	@cd .bingo && $(GO) build -mod=mod  -modfile=golangci-lint.mod -o=$(GOBIN)/golangci-lint-v1.27.0 "github.com/golangci/golangci-lint/cmd/golangci-lint"
+
+OPERATOR_SDK := $(GOBIN)/operator-sdk-v0.18.2
+$(OPERATOR_SDK): .bingo/operator-sdk.mod
+	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
+	@echo "(re)installing $(GOBIN)/operator-sdk-v0.18.2"
+	@cd .bingo && $(GO) build -mod=mod -modfile=operator-sdk.mod -o=$(GOBIN)/operator-sdk-v0.18.2 "github.com/operator-framework/operator-sdk/cmd/operator-sdk"
```

Comment:
so I guess you manually changed it to `-mod=mod` right? What you changed here is not tool module vendoring II think

>  If invoked with -mod=mod, the go command loads modules from the module cache
even if there is a vendor directory present.

So, if you never installed this tool before on some CI with empty cache, those will be fetched I guess?

Is this option changes anything then?

Reply:
Indeed it is, because it tells go build to not look in my vendor dir and use the cache instead.